### PR TITLE
Add compat data for IDBEnvironment

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/IDBEnvironment

### CODE REVIEW

File: api/IDBEnvironment.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
+  "api": {
+    "IDBEnvironment": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBEnvironment",
+        "support": {
+          "webview_android": {
+            "version_added": "4.4"
```
Comment (jpmedley) [inline api/IDBEnvironment.json]: Webview uses Chromium version numbers, not Android numbers. Please change to `"24"`.

File: api/IDBEnvironment.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+              "version_added": "24"
+            },
+            {
+              "version_added": "23",
+              "version_removed": "57",
+              "prefix": "webkit"
+            }
+          ],
+          "chrome_android": {
+            "version_added": "4.4"
```
Comment (jpmedley) [inline api/IDBEnvironment.json]: Chrome Android uses Chromium version numbers, not Android numbers. Please change to `"24"`.

File: api/IDBEnvironment.json (hunk 1 above)
Comment (maboa) [reply api/IDBEnvironment.json]: 👍

File: api/IDBEnvironment.json (hunk 2 above)
Comment (maboa) [reply api/IDBEnvironment.json]: 👍

**Pull request merged**